requirements/edx/base.txt

```diff
@@ -203,4 +203,4 @@ py2neo==3.1.2
 
 # Support for plugins
 web-fragments==0.2.2
-xblock==1.0.0
+git+https://github.com/edx/XBlock@xblock-1.1.0#egg=XBlock==1.1.0
```

Comment:
Is this just temporary, or is there some reason we can't use PyPI for the new version as well?

Reply:
This is temporary.  The latest version is not on PyPI. https://pypi.python.org/pypi/XBlock.